js.js

```diff
@@ -0,0 +1,40 @@
+const books = [];
+const addBtn = document.getElementById('add-new-book');
+const booksoutput = document.querySelector('#booksoutput');
+
+function outputbooklist() {
+  const booklist = JSON.parse(localStorage.getItem('books'));
+  booksoutput.innerHTML = '';
+  booklist.forEach((book, index) => {
+    booksoutput.innerHTML += `<div class = "addedbook"><p>${book.title}</p><p>${book.author}</p><button onclick='removefromlist(${index})'>Remove</button></div`;
+  });
+}
+
+function removefromlist(index) {
+  const booklist = JSON.parse(localStorage.getItem('books'));
+  booklist.splice(index, 1);
+  books.splice(index, 1);
+  localStorage.setItem('books', JSON.stringify(booklist));
+  outputbooklist();
+}
+
+const addBooks = () => {
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+
+  if (title.trim() === '' || author.trim() === '') {
+    return;
+  }
+
+  const newBook = {
+    title,
+    author,
+  };
+
+  books.push(newBook);
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use of array methods such as `concat` over `push`, which doesn't mutate the array.